lib/paper_trail/events/base.rb

```diff
@@ -107,7 +107,7 @@ def attribute_in_previous_version(attr_name, is_touch)
       end
 
       # @api private
-      def changed_and_not_ignored
+      def calculated_ignored_array
```

Comment:
Extracts this calculation to be used by `changed_and_not_ignored` as well as `ignored_attr_has_changed?`